Plugin/ConfigFieldPlugin.php

```diff
@@ -83,15 +83,15 @@ public function afterGetComment(Subject $subject, $result)
             $arguments = $phrase->getArguments();
         }
 
+        if ($resultIsPhrase) {
+            $result = new Phrase($result, $arguments);
```

Comment:
Doesn't this exactly reverse what we did in the lines before?

We are transforming a phrase into text and arguments:
```
            $phrase = $result;
            $result = $phrase->getText();
            $arguments = $phrase->getArguments();
```

And then we are transforming it back to a phrase with the same text and arguments as before:
```
$result = new Phrase($result, $arguments);
```

The original code transformed a phrase into a string, appended another string and then transformed it back to a phrase.

@sebastian-ehrling Can you explain how your change works?

Reply:
Hi Andreas,

thx for your comment. With your plugin for the method getComment the original comments will not be translated anymore. This is because the phrase to translate contains the information about the current config path.

With my modification, we translate the comment phrase before adding the path information.

![path-1](https://user-images.githubusercontent.com/20629208/160923746-c244990f-1d38-4a53-a44a-5a2f2da3a65e.jpg)
![path-2](https://user-images.githubusercontent.com/20629208/160923760-a17d88f4-b0b3-41a7-9e8e-f6861c3fef36.jpg)